utils/colors-vars-map.ts

```diff
@@ -0,0 +1,216 @@
+  },
+  {
+    color: "--shareable-color",
+    description: "Use when you want to indicate that something is shareable (board, dashboard...)"
+  },
+  {
+    color: "--warning-color",
+    description: "Use when you want to indicate a warning action/state (severity, alert, caution...)"
+  },
+  { color: "--warning-color-hover", description: "Use only as hover color on warning color" },
+  { color: "--warning-color-selected", description: "Use as selected indication for warning colors" },
+  {
+    color: "--warning-color-selected-hover",
+    description: "Use this to indicate hover state on a warning-selected items"
+  },
+  // borders
+  { color: "--ui-border-color", description: "Border color for ui elements and components (Button, Input...)" },
+  { color: "--ui-border-on-secondary-color", description: "Border color for ui elements on secondary color" },
+  {
+    color: "--layout-border-color",
+    description: "Border color for general layout and separators (Leftpane, Menu Divider...)"
+  },
+  {
+    color: "--layout-border-on-secondary-color",
+    description: "Border color for general layout on secondary background color"
+  },
+  { color: "--placeholder-color", description: "Use for placeholder text in inputs fields" },
+  {
+    color: "--placeholder-on-secondary-color",
+    description: "Use for placeholder text in inputs fields on secondary background color"
+  },
+  { color: "--icon-color", description: "Default color for icons" },
+  { color: "--icon-on-secondary-color", description: "Default color for icons on secondary background color" },
+  // disabled
+  {
+    color: "--disabled-background-color",
+    description: "Use as background for disabled elements (ui hovers or elements)"
+  },
+  { color: "--disabled-text-color", description: "Use as text in disabled components" },
+  {
+    color: "--disabled-background-on-secondary-color",
+    description: "Use as background for disabled elements on secondary background"
+  },
+  {
+    color: "--disabled-text-on-secondary-color",
+    description: "Use as text in disabled components on secondary background color"
+  },
+  // Link
+  { color: "--link-color", description: "Use only for links" },
+  { color: "--link-on-secondary-color", description: "Use only for links on secondary colors" },
+  // Backgrounds
+  { color: "--primary-background-color", description: "Primary background color" },
+  { color: "--secondary-background-color", description: "Secondary background color" },
+  { color: "--grey-background-color", description: "Grey background color" },
+  { color: "--allgrey-background-color", description: "Grey background color, stays grey in dark and black themes" }
+];
+
+export const colorsHashMap = colorsMap.reduce((map, current) => {
+  const newColorName = current.color.substring(2);
+  map.set(newColorName, current.description);
+  return map;
+}, new Map());
+
+export const contentColors = [
```

Comment:
These content colors are used both for documentation and for actual code (in vibe).
This could lead to duplication and inconsistency.

If they are used not only for documentation - they should be part of the `monday-ui-style` content and consumed from the docs

Reply:
Storybook related tokens are now exported by vibe-storybook-components and are used in all of the stories and components which are used to build the documentation